.travis.yml

```diff
@@ -40,3 +40,41 @@ jobs:
       script:
         - RAILS_ENV=test bundle exec rails db:setup
         - bundle exec rspec
+
+    - stage: test
+      name: 'yarn audit'
+      script:
+        - yarn audit
+
+    - stage: test
+      name: 'bundle audit'
+      script:
+        - gem install bundler-audit
+        - bundle audit update
+        - bundle audit check
+
+    - stage: test
+      name: 'brakeman check'
+      script:
+        - gem install brakeman
+        - brakeman --run-all-checks
+
+  allow_failures:
```

Comment:
Just wondering about the double-up here - the bundle and brakeman audits are repeated in both the include and allow_failures blocks. Why not just include under allow_failures?

Reply:
If you include it only in the `allow_failures` array, the job is not run. The [documentation](https://docs.travis-ci.com/user/customizing-the-build/#rows-that-are-allowed-to-fail) says that you need to have an "exactly" matching job in the `allow_failures`